libs/platform/src/lib/components/link/link.component.scss

```diff
@@ -0,0 +1,10 @@
+@import '~fundamental-styles/dist/link';
+
+.fdp-link-truncate__txt {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: inline-block;
+    vertical-align: text-bottom;
+    max-width: 100%;
+}
```

Comment:
It's not a good idea to have styling in the NGX implementation. If you need this functionality and it does not exist in Fund-styles you can create an issue there so we handle it.

Reply:
need to include fd-ellipsis mixing with other property inside fd-link mixing. need to raise separate PR in fundamental-styles for this.